src/bag3_testbenches/measurement/char/__init__.py

```diff
@@ -216,40 +223,70 @@ def compute_passives(meas_results: Mapping[str, Any], passive_type: str) -> Mapp
     )
     if passive_type == 'cap':
         results['cc'] = estimate_cap(freq, yc)
-        results['r_series'] = np.mean(1 / yc).real
+        results['r_series'] = (1 / yc).mean(axis=-1).real
     elif passive_type == 'res':
         results['c_parallel'], results['res'] = estimate_esd(freq, yc)
-
-        warr = 2 * np.pi * freq
-        z_meas = 1 / (yc + (ypp * ypm) / (ypp + ypm))
-        if results['cpp'] == 0 or results['cpm'] == 0:
+        if np.mean(results['cpp']) == 0 or np.mean(results['cpm']) == 0:
```

Comment:
This matches the prior code, but realistically, the mean will not be zero. We can make this np.close in the future.

Reply:
It is in fact 0 in schematic simulations, but sure, I can change it to `np.close`.